### PR TITLE
Add form, fix lang in correct tag, mongodb

### PR DESCRIPTION
**Pull request by j-mes**
Adding the form in for now, and the correct placement for the lang definition.
Adding mongodb stuff this weekend 😃

### CODE REVIEW

File: server/controllers/post-form-controller.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const main = (req, res) => {
+	console.log('req.body.name:', req.body.name);
+	console.log('req.body.email:', req.body.email);
+	console.log('req.body.country:', req.body.country);
+	console.log('req.body.gdpr:', req.body.gdpr);
```
Comment (lc512k) [inline server/controllers/post-form-controller.js]: A nice feature of ES2015+ is destructuring. You can read about it from Wes here https://wesbos.com/destructuring-objects/

Destructuring your `res` object like:

```js
const main = ({ body: { name, email, country, gdpr }}, res) => {
```
will allow you to reference those attributes directly in the code as:
```
console.log('name:', name);
```

File: server/controllers/post-form-controller.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const main = (req, res) => {
+	console.log('req.body.name:', req.body.name);
```
Comment (lc512k) [inline server/controllers/post-form-controller.js]: I can't tell from the diff, but did you use the express body-parser or similar to get access to the `req.body` object?

**Review by lc512k — APPROVED** [review]
Well done!

File: server/controllers/post-form-controller.js (hunk 1 above)
Comment (j-mes) [reply server/controllers/post-form-controller.js]: Ooh. I'm going to try that out now. I like the idea of writing less code but getting it to do more!

File: server/controllers/post-form-controller.js (hunk 2 above)
Comment (j-mes) [reply server/controllers/post-form-controller.js]: I had used body-parser. Hm I wonder why it's not showing up... I'll re-add it.

**Comment by j-mes** [comment]
Looking into saving responses right now, hopefully I can figure it out!

**Comment by j-mes** [comment]
Almost, almost. Just need to figure out some check to stop form from submitting if the email address is already in the db.

**Pull request merged**